testframework/src/main/java/net/neoforged/testframework/gametest/ExtendedGameTestHelper.java

```diff
@@ -359,6 +360,11 @@ public void assertNotNull(@Nullable Object var, String message) {
         this.assertTrue(var != null, message);
     }
 
+    public <T> T requireNotNull(@Nullable T value, String message) {
```

Comment:
Just use the existing `assertNotNull` instead of forcing things into extra long lines in the three places this new method is being used